introduction.md

```diff
@@ -9,8 +9,8 @@ attribution:
+In 2013, Vitalik Buterin published the [whitepaper](https://ethereum.org/en/whitepaper/) that conceptualized “A Next-Generation Smart Contract and Decentralized Application Platform” – [Ethereum](https://ethereum.org). Initially launched with a proof-of-work (PoW) consensus algorithm in 2015, the [vision](https://ethereum.org/en/eth2/vision/) has always been for Ethereum to become an [energy-efficient](https://spectrum.ieee.org/computing/networks/ethereum-plans-to-cut-its-absurd-energy-consumption-by-99-percent) proof-of-stake (PoS) network. For the first few years, the Ethereum community labored intensely to develop a PoS consensus mechanism that has the desired security and efficiency; going through ideas such as "[Slasher](https://blog.ethereum.org/2014/01/15/slasher-a-punitive-proof-of-stake-algorithm/)" and "[consensus by bet](https://blog.ethereum.org/2015/12/28/understanding-serenity-part-2-casper/)" until eventually settling on “[Casper the Friendly Finality Gadget](https://arxiv.org/pdf/1710.09437.pdf)” in 2017. At that point, focus switched to the transition - how does a community upgrade the heart of a decentralized network while said heart is still beating?
 
-The solution: [The beacon chain](https://ethereum.org/en/eth2/beacon-chain/). The beacon chain is a fully independent PoS network that runs in parallel to the current PoW Ethereum mainnet. By keeping the PoS chain siloed from the main network a ready-to-ship solution is being perfected without risking the flourishing decentralized application platform of the Etherum PoW chain. A one-way bridge from the PoW network to the PoS network began accepting deposits (as of November 2020) and the beacon chain is currently [live](https://beaconcha.in/), ready for action, and secured by millions of ETH deposited across over 100K validators. Since its launch, in December 2020 the beacon chain has been an absolute success — finalizing 100% of its epochs with no downtime.
+The solution: [The beacon chain](https://ethereum.org/en/eth2/beacon-chain/). The beacon chain is a fully independent network which has a PoS consensus layer. It is running in parallel to the current Ethereum mainnet, where the consensus layer currently remains PoW. By keeping the PoS chain isolated from the main network a ready-to-ship solution is being perfected without risking the flourishing decentralized application platform of the Ethereum PoW chain. A one-way bridge from the PoW network to the PoS network began accepting deposits (as of November 2020) and the beacon chain is [currently](https://beaconscan.com/) [live](https://beaconcha.in/), ready for action and secured by millions of ETH deposited across over 100K validators. Since its launch in December 2020 the beacon chain has been an absolute success — finalizing 100% of its epochs with no downtime.
```

Comment:
The solution: [The beacon chain](https://ethereum.org/en/eth2/beacon-chain/). The beacon chain is a fully independent network which has a PoS consensus layer. It is running in parallel to the current Ethereum mainnet, where the consensus layer currently remains PoW. By keeping the PoS chain isolated from the main network a ready-to-ship solution is being perfected without risking the flourishing decentralized application platform of the Ethereum PoW chain. A one-way bridge from the PoW network to the PoS network began accepting deposits (as of November 2020) and the beacon chain is [currently](https://beaconscan.com/) [live](https://beaconcha.in/), ready for action and secured by millions of ETH deposited across over 240K validators. Since its launch in December 2020 the beacon chain has been an absolute success — finalizing 100% of its epochs with no downtime.